### DISCUSSION TITLE
[Crystools ERROR]

### DISCUSSION THREAD

**Discussion by lambbigcc** [crystian/ComfyUI-Crystools#60] [Q&A] [answered]
[Crystools ERROR] For some reason, pynvml is not working in a laptop with only battery, try to connect and turn on the monitor
[Crystools ERROR] Monitor of GPU is turning off (not on UI!)
My laptop graphics card is 4090 and it is connected to the power supply, the above problem is displayed when running comfyui, please how can I fix this problem, thanks
![image](https://github.com/crystian/ComfyUI-Crystools/assets/19486651/2825f950-77ab-44b2-81c7-2a8e58613dba)

**Comment by crystian** [comment, marked as answer]
It is from the driver; I can't do anything. Sorry :(

**Comment by bdjiaoksjs1** [comment]
更新一下驱动程序可能会解决
